### DISCUSSION TITLE
What to do after cloning a repository succesfully ?

### DISCUSSION THREAD

**Discussion by githubYannick** [community/community#101432] [New to GitHub]
Hey,
so i am pretty new to pragramming and git/ github but i have to use it for work and im currently stuck in an never ending circle of problems!

1) I want to use a certain repository from github in my android studio project. 
2) the problem is i work on 2 different computers. On one of those, my home computer, i can easily acces the packages from this repository. On my work computer and my laptop i sadly cannot get an acces to those. it gives me the "unresolved reference" error. 

I tried all the solutions i could think of, but none of them worked. i tried to redo everything how i did it on my home computer, but somehow it doesnt work.

So i guess my question is: 

**what shall i do, after cloning a repository, to acces packages from this repository?**

In my research i couldnt find an answer that satifies my specific problem and/or didnt solve it. 

If you need my code/ pictures of the errors etc. please let me know. because this is my first question ever im not sure how much you guys would need to help me.

Thanks in advance! :)

**Comment by Chupriarti** [comment]
What repository are you talking about? And what platforms (Windows or Mac) are used?

**Reply by githubYannick** [reply]
Im using windows and this is the link towards the repository: https://github.com/mik3y/usb-serial-for-android

In the "readMe" is an follow up instruction for gradle settings which i followed as well.

**Comment by sonersyln** [comment]
After cloning a repository, you should be able to access the packages from this repository by adding the necessary dependencies in your project's build.gradle file. Here's a general step-by-step guide:

Clone the Repository: Use the command git clone <repository_url> in your terminal to clone the repository.

Open Project in Android Studio: After cloning, open the project in Android Studio.

Update Dependencies: Open your project's build.gradle file and add the necessary dependencies for the packages you want to use. The specific dependencies you need to add will depend on the packages in the repository you cloned.

Sync Project: After adding the dependencies, click on 'Sync now' or 'Sync Project with Gradle Files' in Android Studio.

Import Packages: Now, you should be able to import and use the packages in your code.
